viewer/packages/cad-styling/src/SinglePropertyFilterNodeCollection.test.ts

```diff
@@ -3,41 +3,23 @@
  */
 
 import { CogniteClient } from '@cognite/sdk';
-import { Cognite3DModel } from '../../../public/migration/Cognite3DModel';
 
 import { SinglePropertyFilterNodeCollection } from './SinglePropertyFilterNodeCollection';
-import { CadMaterialManager, CadNode } from '@reveal/rendering';
-import { CadModelMetadata } from '@reveal/cad-parsers';
-import { NodesApiClient, NodesLocalClient } from '@reveal/nodes-api';
 import { IndexSet, NumericRange } from '@reveal/utilities';
 
-import { createCadModelMetadata, generateV8SectorTree } from '../../../../../test-utilities';
-
 import nock from 'nock';
-import { V8SectorRepository } from '@reveal/sector-loader';
-import { Mock } from 'moq.ts';
+import { CdfModelNodeCollectionDataProvider } from './CdfModelNodeCollectionDataProvider';
 
 describe('SinglePropertyFilterNodeCollection', () => {
   let set: SinglePropertyFilterNodeCollection;
   let client: CogniteClient;
-  let model: Cognite3DModel;
+  let model: CdfModelNodeCollectionDataProvider;
   const filterNodesEndpointPath = /.*\/nodes\/list/;
 
   beforeEach(() => {
     client = new CogniteClient({ appId: 'test', project: 'dummy', getToken: async () => 'dummy' });
 
-    const mockV8SectorRepository = new Mock<V8SectorRepository>();
-    const materialManager = new CadMaterialManager();
-
-    const cadModelMetadata: CadModelMetadata = createCadModelMetadata(8, generateV8SectorTree(3, 3));
-
-    materialManager.addModelMaterials(cadModelMetadata.modelIdentifier, 0);
-
-    const cadNode: CadNode = new CadNode(cadModelMetadata, materialManager, mockV8SectorRepository.object());
-    const nodesClient: NodesApiClient = new NodesLocalClient();
-
-    model = new Cognite3DModel(1, 2, cadNode, nodesClient);
-    model = { modelId: 112, revisionId: 113 } as Cognite3DModel;
+    model = { modelId: 112, revisionId: 113 } as CdfModelNodeCollectionDataProvider;
     set = new SinglePropertyFilterNodeCollection(client, model);
   });
```

Comment:
Hm... It's almost like... Abstractions make testing easier 🤔

👏👏👏👏👏👏👏👏👏👏👏👏👏👏

Reply:
Turns out `model` was assigned twice, and the first (and unused one) was the one that required most imports :P